.github/workflows/docker.yml

```diff
@@ -1,5 +1,9 @@
 name: Build Docker Images
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 */7 * *'
```

Comment:
No, this is wrong. You should consider `0 0 * * 1`, which builds every Monday. `*/7` for day of the month is inaccurate since the number of days in a month is only divisible by 7 in February of a non-leap year.